src/hotspot/os/linux/crac_linux.cpp

```diff
@@ -327,6 +329,15 @@ bool VM_Crac::check_fds() {
       }
     }
 
+    // On some systems using SSSD files in this directory are left open
+    // after calling getpwuid_r, getpwname_r, getgrgid_r, getgrname_r
+    // or other functions in this family.
+#define SSS_NSS_MEMCACHE "/var/lib/sss/mc/"
+    if (!strncmp(details, SSS_NSS_MEMCACHE, sizeof(SSS_NSS_MEMCACHE) - 1)) {
```

Comment:
I believe GCC can manage with `const char *` and `strlen` call in compile time here. `constexpr` could be addedaswell, but nor necessary I guess.

Reply:
Right; this is not the most perf-critical part but for the fun I've checked the assembly and since this is 16 bytes it is easily replaced with 2 64-bit constant comparisons.
Btw. for some reason the compiler emits warning for just `constexpr char *...`, I have to write `constexpr const char *...`.